Confus/Confus/Flag.h

```diff
@@ -1,69 +1,60 @@
 #pragma once
-namespace irr 
-{
-    class IrrlichtDevice;
-    namespace scene {
-        class IAnimatedMeshSceneNode;
-    }
-}
-
-namespace Confus 
-{
-	class Player;
-}
 
 namespace Confus 
 {
+class Player;
+class Collider;
 // Team Identifier
 /// <summary>
 /// The Team's Identifier. A player has a team, flag has a team, ui has a team, etc.
 /// </summary>
 enum class ETeamIdentifier 
 {
 	/// <summary>This object doesn't have a team identifier.</summary>
-	NONE,
+	None,
 	/// <summary>This object has a red team identifier.</summary>
-	TEAM_RED,
-	/// <summary>This object has a green team identifier.</summary>
-	TEAM_BLUE
+	TeamRed,
+	/// <summary>This object has a blue team identifier.</summary>
+	TeamBlue
 };
 enum class EFlagEnum
 { 
-    NONE,
-    FLAG_BASE,
-    FLAG_TAKEN,
-    FLAG_DROPPED 
+    None,
+    FlagBase,
+    FlagTaken,
+    FlagDropped 
 };
 
 	//Flag Class, every flag should have this class, contains info about a flag
```

Comment:
Changed it accordingly to: [http://stackoverflow.com/questions/13752191/documenting-enum-values-with-doxygen](http://stackoverflow.com/questions/13752191/documenting-enum-values-with-doxygen).